tx-source/site_core.php

```diff
@@ -1210,6 +1210,7 @@
 $Definition['Position'] = 'Position';
 $Definition['Post'] = 'Post';
 $Definition['post'] = 'post';
+$Definition['Notifications Postmaster'] = 'Notifications Postmaster';
```

Comment:
As I know we are trying to keep values sorted alphabetically in this file

Reply:
That's right.